src/Data/Food/Builder/Recipe.elm

```diff
@@ -192,9 +191,7 @@ compute db =
                                         )
                                     |> Maybe.withDefault (Transport.default Impact.noImpacts)
                         in
-                        db.processes
-                            |> Process.loadWellKnown
-                            |> Result.map (Transport.computeImpacts db.impacts mass transport)
+                        Transport.computeImpacts db.impacts mass transport db.wellKnown
```

Comment:
This function could just accept a `db` argument and extract `impacts` and `wellKnown` from it.

Reply:
Good point indeed, but for the signature i'll need to use extensible records notation `{ impacts : List Definition, wellKnown : Process.WellKnown }` as I can't import the Db in this file, or it'll create an import loop between `src/Data/Transport.elm` and `src/Data/Food/Db.elm`.